databricks/koalas/base.py

```diff
@@ -323,3 +328,126 @@ def notnull(self):
         return ~self.isnull()
 
     notna = notnull
+
+    # TODO: axis, skipna, and many arguments should be implemented.
+    def all(self, axis: int = 0) -> bool:
+        """
+        Return whether all elements are True.
+
+        Returns True unless there at least one element within a series that is
+        False or equivalent (e.g. zero or empty)
+
+        Parameters
+        ----------
+        axis : {0 or 'index'}, default 0
+            Indicate which axis or axes should be reduced.
+
+            * 0 / 'index' : reduce the index, return a Series whose index is the
+              original column labels.
+
+        >>> ks.Series([True, True]).all()
+        True
+
+        >>> ks.Series([True, False]).all()
+        False
+
+        >>> ks.Series([0, 1]).all()
+        False
+
+        >>> ks.Series([1, 2, 3]).all()
+        True
+
+        >>> ks.Series([True, True, None]).all()
+        True
+
+        >>> ks.Series([True, False, None]).all()
+        False
+
+        >>> ks.Series([]).all()
+        True
+
+        >>> ks.Series([np.nan]).all()
+        True
+        """
+
+        if axis not in [0, 'index']:
+            raise ValueError('axis should be either 0 or "index" currently.')
+
+        sdf = self._kdf._sdf.select(self._scol)
+        col = self._scol
+
+        # Note that we're ignoring `None`s here for now.
+        # any and every was added as of Spark 3.0
+        # ret = sdf.select(F.expr("every(CAST(`%s` AS BOOLEAN))" % sdf.columns[0])).collect()[0][0]
+        # Here we use min as its alternative:
+        ret = sdf.select(F.min(F.coalesce(col.cast('boolean'), F.lit(True)))).collect()[0][0]
```

Comment:
and, this uses min for all.

Reply:
Shouldn't we use `False` as the default value here?

Reply:
it uses Pandas's behaviour .. (and appearntly that's what current `every` in Spark does too ..)

```
[null] -> true
[null, true] -> true
[true] -> true
[] -> true
```
```
[False] -> False
[null, False] -> False
```